scrollingpagerindicator/src/main/java/ru/tinkoff/scrollingpagerindicator/ScrollingPagerIndicator.java

```diff
@@ -538,7 +539,12 @@ protected void onDraw(Canvas canvas) {
 
                 paint.setColor(calculateDotColor(scale));
                 if (orientation == LinearLayoutManager.HORIZONTAL) {
-                    canvas.drawCircle(dot - visibleFramePosition,
+                    float cx = dot - visibleFramePosition;
+                    if (isRtl()) {
+                        cx = getWidth() - cx;
```

Comment:
It will work not correct with first version of ViewPager. ViewPager1 does not support RTL.

Seems like we need to make it possible to turn off RTL mode by providing some method for users of this lib.

Reply:
@Stasssm 
Thanks for your note. I added an option for those users to be able to disable this feature.